### PR TITLE
update utils

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

- update to latest utils
- also update types-python-dateutil

# Test instructions | Instructions pour tester la modification

- poke around in admin PR app

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -49,11 +49,10 @@ unidecode = "^1.3.6"
 
 # PaaS
 awscli-cwlogs = "^1.4.6"
-notifications-utils = { git = "https://github.com/cds-snc/notifier-utils.git", tag = "52.2.8" }
+notifications-utils = { git = "https://github.com/cds-snc/notifier-utils.git", tag = "52.3.2" }
 
 # Pinned dependencies
 rsa = "^4.1" # not directly required, pinned by Snyk to avoid a vulnerability
-certifi = "^2023.7.22" # Pinned for security reasons: https://github.com/cds-snc/notification-admin/security/dependabot/169
```
Comment (sastels) [inline pyproject.toml:56]: utils now requires a later (and also secure) version

**Review by ben851 — APPROVED** [review]
LGTM

**Review by P0NDER0SA — APPROVED** [review]
yep!

**Comment by sastels** [comment]
Done by Will (and to a later utils) in #1941

**Pull request closed without merging**